src/gui/GuiRunner.cc

```diff
@@ -151,25 +151,10 @@ void GuiRunner::RequestState()
 }
 
 /////////////////////////////////////////////////
-void GuiRunner::OnPluginAdded(const QString &_objectName)
+void GuiRunner::OnPluginAdded(const QString &)
```

Comment:
Can this be removed since this is targeting `main`?

Reply:
This function may make a comeback on Fortress, in https://github.com/ignitionrobotics/ign-gazebo/pull/793, so I'd rather leave it there until we sort that out.